### ISSUE TITLE
[Doc] Aadd a workaroud about how to solve rancher-monitoring-crd-manager is missing

### ISSUE THREAD

**Issue by w13915984028** [harvester/harvester#5505]
**Is your doc request related to a problem? Please describe or add related issue ID.**

Such a message is seen:

```
kubectl get bundles -A
NAMESPACE     NAME                                          BUNDLEDEPLOYMENTS-READY   STATUS
fleet-local   fleet-agent-local                             1/1                       
fleet-local   local-managed-system-agent                    1/1                       
fleet-local   mcc-harvester                                 1/1                       
fleet-local   mcc-harvester-crd                             1/1                       
fleet-local   mcc-local-managed-system-upgrade-controller   1/1                       
fleet-local   mcc-rancher-logging-crd                       1/1                       
fleet-local   mcc-rancher-monitoring-crd                    0/1                       Modified(1) [Cluster fleet-local/local]; clusterrole.rbac.authorization.k8s.io rancher-monitoring-crd-manager missing; clusterrolebinding.rbac.authorization.k8s.io rancher-monitoring-crd-manager missing; configmap.v1 cattle-monitoring-system/rancher-monitoring-crd-manifest missing; serviceaccount.v1 cattle-monitoring-system/rancher-monitoring-crd-manager missing

```

**Describe the solution you'd like**

Add a workaroud about how to solve it.

**Comment by w13915984028** [comment]
FYI:

The root cause may be related to the `rancher-monitoring-crd-manager` ClusterRole in old Harvester versions (v1.1.*), which has no `update` operation, on the upgrade path, the new CRD charts was not applied effectively (via `update` ?) and hence a couple of objects are missing the required `annotations` and `labels`.

(not verified yet)

When we remove & create `rancher-monitoring-crd` managedchart, it can recover. But the removing of CRD chart will cause users' CRD objects are also removed. A better [workaround](https://github.com/harvester/docs/pull/541) is added to patch the related objects accordingly. 

```
$ kubectl get clusterrole rancher-monitoring-crd-manager
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  annotations:
    meta.helm.sh/release-name: rancher-monitoring-crd
    meta.helm.sh/release-namespace: cattle-monitoring-system
  creationTimestamp: "2023-01-09T11:04:33Z"
  labels:
    app: rancher-monitoring-crd-manager
    app.kubernetes.io/managed-by: Helm
  name: rancher-monitoring-crd-manager
  ...
rules:
- apiGroups:
  - apiextensions.k8s.io
  resources:
  - customresourcedefinitions
  verbs:
  - create
  - get
  - patch
  - delete
```

**Comment by w13915984028** [comment]
Harvester document is updated.

**Comment by bpedersen2** [comment]
I came across this as well ( after/during an upgrade to 1.4.1). The 'manual' workaroud did unblock the repo, but caused a subsequent similar error on the mcc-harvester bundle. 

I was able to fully get it fixed by adding the the new [1] correctDrift.enabled to true in the bundles (and then after successful reconciliation remove it again). 

I guess it would make sense to  add that flag to the charts by default


[1] https://github.com/rancher/fleet/pull/1594
